01-basics/03-MarkedList/script.js

```diff
@@ -29,4 +29,28 @@ const emails = [
   'Isaias_Kuhic@jarrett.net',
 ];
 
-// Требуется создать Vue приложение
+const app = createApp({
+  emails,
```

Comment:
Adding a constant to options only makes sense if the constant must be accessible from instance properties, e.g. inside the template. Otherwise, just a JS variable can be used as usual.